argo-cd-apps/base/member/infra-deployments/workspaces/workspaces.yaml

```diff
@@ -27,7 +27,6 @@ spec:
         repoURL: https://github.com/redhat-appstudio/infra-deployments.git
         targetRevision: main
       destination:
-        namespace: workspaces-system
```

Comment:
Removing this is the right thing to do but we need to make sure none of the NS resources are missing NS name in the metadata.

I checked and only the 2 following resources are relying on Argo to set NS. Please set then NS explicitly on them as part of this change:

- https://github.com/redhat-appstudio/infra-deployments/blob/main/components/workspaces/team/base/workspaces-team-role.yaml#L1
- https://github.com/redhat-appstudio/infra-deployments/blob/main/components/workspaces/team/base/workspaces-team-rolebinding.yaml#L1

Reply:
Got it

Reply:
those resources are not needed any longer, removing them